frontend/javascripts/oxalis/controller/td_controller.js

```diff
@@ -161,42 +168,40 @@ class TDController extends React.PureComponent<Props> {
         this.setTargetAndFixPosition();
       },
       pinch: delta => this.zoomTDView(delta, true),
+      mouseMove: (delta: Point2, position: Point2) => {
+        Store.dispatch(setMousePositionAction([position.x, position.y]));
```

Comment:
Previously, this was only done when no skeleton controller existed. I think, the rationale behind this was that we wouldn't need the mouse position then (because the mapping view wouldn't exist). However, updating the position in all cases seems less brittle to me.

Reply:
Do you mean this was only done when a volume controller existed? Because the existence of the mapping view is not related to the existence of a skeleton. If your description is correct, I don't understand how this could've worked before (or I'm misunderstanding :see_no_evil:)

Reply:
I had another look in the code and my example with the mapping view was a bit wrong. This particular mouse move handler is only responsible for updating the mouse position when moving the mouse in the 3d viewport. And that position is only used for one thing as far as I can see: for isosurface hit tests (i.e., for highlighting and for jumping to isosurfaces). That explains that only the isosurface-related functionality was broken in certain scenarios.

However, in both of the above cases the hit-test is triggered by an event (which contains the mouse position). Therefore, I removed the dispatch of the updateMousePosition-action and instead pass the mouse position directly into the hit test. This also removes the (throttled) call of `performIsosurfaceHitTest` which was done in every render call of the plane view.

Could you have a another look at the latest commit? :)

Reply:
Ah, I see, thank you for having another look! The changes make sense to me :+1: